zulipterminal/ui_tools/views.py

```diff
@@ -1134,11 +1137,13 @@ def __init__(self, controller: Any, stream_id: int) -> None:
         user_ids = model.get_other_subscribers_in_stream(stream_id=stream_id)
         user_names = [model.user_name_from_id(id) for id in user_ids]
         sorted_user_names = sorted(user_names)
-        # sorted_user_names.insert(0, model.user_full_name)
+        sorted_user_names.insert(0, model.user_full_name)
+        total_members = len(sorted_user_names)
         title = 'Stream Members (up/down scrolls)'
 
-        stream_users_content = [('', [
-            (name, '') for name in sorted_user_names]
+        stream_users_content = [(
+            'Subscribed members({})'.format(total_members), [
```

Comment:
I'm not sure the members list needs a category, as there's just a list of users.

For the number of members, we can place this in the stream info popup, before the `Press m to view` text.